### DISCUSSION TITLE
New spanish student

### DISCUSSION THREAD

**Discussion by Eduu84** [Cyfrin/foundry-full-course-cu#1553] [General]
Hi!

I just finished the blockchain basics course, although I am not a beginner, it has been very useful to refresh things and learn others that I did not know.

I am a Spanish guy, with a lot of desire to learn blockchain programming in general. 

The hardest thing for me is that all the good courses like this one are in English.

I start with a lot of desire both to learn to program and to learn to speak English.

And excuse me if my writing in English is not correct, remember, I'm learning :)

**Comment by EngrPips** [comment]
Hello @Eduu84 , so glad that your desire to learn overshadow the language barrier. Just keep going and with time it will get more easy

**Reply by Eduu84** [reply]
Thank you very much for your encouragement, sooner or later I will learn it.

**Reply by EngrPips** [reply]
Indeed!
